### PR TITLE
cleanup readme for clear differences between prod and local test deployment

### PR DESCRIPTION
**Pull request by akash-roboticist**
### Fixed bug

Fixed docker image structure diagram

### Fix applied

**If your pull request is a bug fix, delete this line and all the below lines.**

**If your pull request is a feature implementation, delete this line and all the above lines.**

### Implemented feature

Re-formatted readme with rollup markdown to minimize confusions between steps for prod vs local deployment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,17 +6,49 @@ well as on-prem environments
 
 ![](../media/rmf_banner.png?raw=true)
 
-## The Kubernetes way of deployment
+<details>
+<summary> ** Quick local deployment for testing, in lieu of kubernetes cluster **</summary>
```
Comment (aaronchongth) [inline README.md:10]: Quick check, is this meant to be bolded text? Or are the double asterisks intended

File: README.md [hunk 2]
```diff
@@ -185,8 +194,10 @@ Or specify the location of the kubeconfig file in the command:
 kubectl --kubeconfig /etc/rancher/k3s/k3s.yaml get pods --all-namespaces
 helm --kubeconfig /etc/rancher/k3s/k3s.yaml ls --all-namespaces
 ```
-### Services
+</details>
 
+<details>
+<summary>Services list</summary>
 List of ports and URIs used by the different services:
```
Comment (aaronchongth) [inline README.md:201]:
```suggestion
<summary>List of ports and URIs used by the different services</summary>
```
We can probably make the summary more explicit and forgo the next line inside the details

**Review by aaronchongth — APPROVED** [review]
Left a couple of comments but overall LGTM!

File: README.md (hunk 2 above)
Comment (akash-roboticist) [reply README.md:201]: Good suggestion!

File: README.md (hunk 1 above)
Comment (akash-roboticist) [reply README.md]: Intended this to be bold, and wanted to correct the syntax, thanks for catching this. fixed in https://github.com/open-rmf/rmf_deployment_template/pull/41/commits/d8ea2450dab422aebe93fc8143ff796e3270c4ff

**Pull request merged**